core/src/test/scala/za/co/absa/spline/harvester/plugin/embedded/SnowflakePluginSpec.scala

```diff
@@ -0,0 +1,58 @@
+package za.co.absa.spline.harvester.plugin.embedded
```

Comment:
Missing copyright/license header. That's the reason why all test builds fail.

Reply:
Adding this!